src/main/java/com/revify/service/FeedbackserviceImpl.java

```diff
@@ -0,0 +1,163 @@
+package com.revify.service;
+
+import com.revify.dto.*;
+import com.revify.entity.*;
+import com.revify.repository.FeatureReviewRepository;
+import com.revify.repository.PurchasedProductUserRepository;
+import com.revify.repository.ReviewRepository;
+import com.revify.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Timestamp;
+import java.util.*;
+
+/**
+ * Created by MohulT430 on 4/4/2015.
+ */
+@Service
+public class FeedbackserviceImpl implements FeedbackService {
```

Comment:
FeedbackserviceImpl should be renamed to FeedbackServiceImpl; both class and filename